src/include/brain/brain.h

```diff
@@ -16,6 +16,9 @@
 #include <cstdio>
 #include <utility>
 #include <functional>
+#include <pqxx/pqxx>
+#include "settings/setting_id.h"
+#include "settings/settings_manager.h"
```

Comment:
This .h file is going to be used in many places. So I think we should move the implementation of `BrainEnvironment()` and these two includes into .cpp file to reduce dependency.